cypress/integration/frontsite/literatureRequest.test.spec.js

```diff
@@ -0,0 +1,34 @@
+describe('literature request form', () => {
+  Cypress.Cookies.defaults({
```

Comment:
Instead of redefining this, Cypress has support for [global-default-cookies](https://docs.cypress.io/api/cypress-api/cookies.html#Set-global-default-cookies)